packages/arb-ts/token_lists/generate_json.ts

```diff
@@ -0,0 +1,114 @@
+import { instantiateBridge } from '../scripts/instantiate_bridge'
+import uniswapDefaultList from './uniswapDefaultList'
```

Comment:
We could define the json import as a user env variable (or cli arg) to make the script more flexible

Reply:
import json in js wants a string literal. I'm sure there are ways around this lol, but I'm gonna leave as is for now if that's cool